app/src/main/AndroidManifest.xml

```diff
@@ -53,6 +54,7 @@
                 android:name="android.accessibilityservice"
                 android:resource="@xml/autofill_config" />
         </service>
+        <service android:name=".ClipboardService" />
```

Comment:
This can be probably moved to a separate process: [service-element](https://developer.android.com/guide/topics/manifest/service-element)

Reply:
Explain pl0x

Reply:
> Explain pl0x

https://github.com/haroldadmin/WhatTheStack/blob/master/what-the-stack/src/main/AndroidManifest.xml#L6